assets/templates/partials/calendar/filter/census.tmpl

```diff
@@ -0,0 +1,41 @@
+<fieldset class="ons-fieldset">
+  <!-- <legend class="ons-fieldset__legend">Content type</legend> -->
+  <!-- <p class="ons-checkboxes__label">Show only:</p> -->
+  <div class="ons-checkboxes__items">
+    <span class="ons-checkboxes__item ons-checkboxes__item--no-border">
+      <span class="ons-checkbox ons-checkbox--no-border">
+        <input type="checkbox" id="publications" class="ons-checkbox__input ons-js-checkbox  ons-js-other ons-js-select-all-children" value="publications" aria-controls="publications-other-wrap" aria-haspopup="true">
+        <label class=" ons-checkbox__label " for="publications" id="publications-label">Census (79)</label>
+        <span class="ons-checkbox__other" id="publications-other-wrap">
+          <fieldset class="ons-fieldset ons-js-other-fieldset">
+            <legend class="ons-fieldset__legend ons-u-vh ons-u-mb-xs">Census year</legend>
+            <p class="ons-checkboxes__label"></p>
+            <div class="ons-checkboxes__items">
+              <span class="ons-checkboxes__item ons-checkboxes__item--no-border">
+                <span class="ons-checkbox ons-checkbox--no-border">
+                  <input type="checkbox" id="datasets" class="ons-checkbox__input ons-js-checkbox " value="datasets">
```

Comment:
Few instances with an extra space at the end of the css classes, are you injecting some in via JS?